middle-pgsql.cpp

```diff
@@ -1263,21 +1263,21 @@ struct pthread_thunk {
 extern "C" void *pthread_middle_pgsql_stop_one(void *arg) {
     pthread_thunk *thunk = static_cast<pthread_thunk *>(arg);
     return thunk->obj->pgsql_stop_one(thunk->ptr);
-};
+}
 } // anonymous namespace
 
 void middle_pgsql_t::stop(void)
 {
     int i;
 #ifdef HAVE_PTHREAD
-    pthread_t threads[num_tables];
+    std::vector<pthread_t> threads(num_tables);
```

Comment:
I don't think there is much to gain here from using a std::vector, as the array is completely static and just local anyways. Doesn't hurt much either, so feel free to ignore me. Dito below.